package.json

```diff
@@ -23,21 +23,21 @@
       "import": "./dist/js/uswds.min.js",
       "default": "./dist/js/uswds.min.js"
     },
-    "./src/js/components": "./src/components/index.js",
-    "./src/js/components/*": "./src/js/components/*.js",
-    "./src/fonts/*": "./src/fonts/*",
-    "./src/img/*": "./src/img/*",
-    "./js/*": "./dist/js/*",
+    "./src/js/components": "./packages/uswds-core/src/js/index.js",
+    "./src/js/components/*": "./packages/*.js",
+    "./src/fonts/*": "./packages/uswds-core/src/assets/fonts/*",
+    "./src/img/*": "./packages/usa-icon/src/img",
+    "./js/*": "./packages/*.js",
     "./css/*": "./dist/css/*",
     "./scss/*": "./dist/scss/*",
```

Comment:
⚠️Breaking change

Due to updated paths `"./scss/*": "./dist/scss/*"` requires includePaths in webpack. Which would mean users would have to eject their app and include the paths in their config.

Reply:
**Needs further testing** 

I've been able to follow this [create-react-app guidance](https://create-react-app.dev/docs/adding-a-sass-stylesheet) to add some include paths with mixed success.

```.env
// .env
SASS_PATH=./node_modules:./node_modules/@uswds:./node_modules/@uswds/uswds/packages
```

```scss
// my-stylesheet.scss
@use "uswds-core" as *;

body {
  background-color: color("green-40v");
}
```